### PR TITLE
[HTML 5] Rounds upload percentage, localize messages in presentation uploader

### PR DESCRIPTION
**Pull request by Tainan404**
- rounds the value of the upload percentage in the presentation uploader
- localize presentation manager's messages

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx [hunk 1]
```diff
@@ -34,69 +34,67 @@ const defaultProps = {
...
+    description: 'message use on the file browser',
   },
   fileToUpload: {
     id: 'app.presentationUploder.fileToUpload',
-    defaultMessage: 'To be uploaded...',
+    description: 'message used in the file selected for upload',
   },
   uploadProcess: {
     id: 'app.presentationUploder.upload.progress',
-    defaultMessage: 'Uploading ({progress}%)',
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx]: Please confirm that the `{progress}` is not needed anymore?  - same with `{current}` `{total}`, etc

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx]: To me it looks like it's needed - see ` progress: Math.floor(item.upload.progress).toString(),`

Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx]: I think @Tainan404 just removed the `defaultMessage` and moved to the `locale/en`

Comment (antobinary) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx]: OK! I see now! :+1:

**Pull request merged**